src/settings/MappingProfiles/detailsSections/edit/OrderDetailSection/OrderInformation.js

```diff
@@ -482,20 +483,20 @@ const OrderInformationComponent = ({
 OrderInformationComponent.manifest = Object.freeze({
   purchaseOrderLinesLimitSetting: {
     type: 'okapi',
-    path: 'configurations/entries?query=(module==ORDERS and configName==poLines-limit)',
+    path: 'orders-storage/settings?query=(key==poLines-limit)',
```

Comment:
This implies a dependency on `orders-storage.settings`, but this is absent from `package.json::stripes.okapiInterfaces`.